libs/design/DataInput/DateInput.tsx

```diff
@@ -0,0 +1,32 @@
+import 'dayjs/locale/fa';
```

Comment:
I guess we should have this import within the entry point of the application instead of this component

Reply:
U mean in `vite.config.ts`?

Reply:
No in `main.tsx`